app/frontend/src/components/TabBar/TabBar.tsx

```diff
@@ -29,20 +31,23 @@ export default function TabBar<T extends Record<string, React.ReactNode>>({
   return (
     <TabList
       aria-label={ariaLabel}
+      value={value}
```

Comment:
Why was this change needed? Is it to allow `undefined` values so you can have no tabs selected?

Reply:
Ah yes, I had seen your commit where you simplified the tabs component. However, I was unable to open a specific tab after that change.

(Example value is 'home' when url is user/home)

So I added the value prop again. Or is there another way to render the tabbar component with a specific tab opened instead of the first one (default)?

Reply:
That should be controllable through the value prop passed to `TabContext`. I just tried it on `develop` by changing the initial state passed to it in the current code without this change and it works, so you should be able to do something similar by deriving the initial state based on the route?

Reply:
That should be controllable through the value prop passed to `TabContext`. I just tried it on `develop` by changing the initial state passed to it in the current code without this change and it works, so you should be able to do something similar by deriving the initial state based on the route?

Reply:
Yes, that works indeed! :)